RFC/src/RFC-0130_Mining.md

```diff
@@ -1 +1,118 @@
+
+This process will ensure that the Tari difficulty remains independent. Adjusting the difficulty will ensure that the Tari block times are preserved. Also, the Tari block time is (hard fork) flexible and can be less than, equal or greater than the Monero block times. A more detailed description of the Merged Mining process between a Primary and Auxiliary blockchain is provided in the [Merged Mining TLU report] (https://tlu.tarilabs.com/merged-mining/merged-mining.html).
+
+#### Functionality required by the Tari Mining Server
+
+- The Tari blockchain MUST have the ability to be merged mined with Monero. 
+- The Mining Server MUST maintain a local or remote connection with a Base Node and a Monero Node.
+- It MUST have a mechanism to construct a new Tari and Monero block by selecting transactions from the different Tari and Monero mempools that need to be included in the different blocks.
+- It MAY have a configurable transaction selection mechanism for the block construction process. 
+- It MAY have the ability to re-verify transactions before including them in a new Tari block.
+- It MUST have the ability to include the block header hash of the new Tari block into the coinbase section of a newly created Monero block to enable merged mining.
+- It MUST be able to include the Monero block header hash, Merkel tree branch and hash of the coinbase transaction of the Monero block into the PoW summary field of the new Tari block header. 
+- It MUST have the ability to transmit and distribute PoW tasks for the newly created Monero block, that contains the Tari block information, to connected Mining Workers.
+- It MUST verify PoW solutions received from Mining Workers and it MUST reject and discard invalid solutions or solutions that do not meet the minimum required difficulty.
+- The Mining Server MAY keep track of mining share contributions of the connected Mining Workers. 
+- It MUST submit completed Tari blocks to the Tari Base Node.
+- It MUST submit completed Monero blocks to the Monero Network.  
+
+#### Functionality required by the Tari Mining Worker
+
+- It MUST maintain a local or remote connection to a Mining Server.
+
+- It MUST have the ability to receive PoW tasks from the connected Mining Server. 
+
+- It MUST have the ability to perform the latest released version of Monero's PoW algorithm on the received PoW tasks.
+
+- It MUST attempt to solve the PoW algorithm at the Tari and/or Monero difficulties. 
```

Comment:
Isn't it possible that Worker nodes get much lower difficulties if they're part of a large pool?

Reply:
Your comment is true, allowing Mining Workers to send lower difficulty solutions to the Mining Server is a method used to track mining share contributions in a pool.
The item was changed to "It MUST attempt to solve the PoW task at the difficulty specified by the Mining Server. "